python/mxnet/util.py

```diff
@@ -848,3 +848,20 @@ def setenv(name, value):
     """
     passed_value = None if value is None else c_str(value)
     check_call(_LIB.MXSetEnv(c_str(name), passed_value))
+
+def set_flush_denorms(value):
+    """Change floating-point calculations when dealing with denormalized values.
+       This is only applicable to architectures which supports flush-to-zero.
+       Denormalized values are positive and negative values that are very close to 0
+       (exponent is the smallest possible value).
+       Flushing denormalized values to 0 can speedup calculations if such values occurs,
+       but if fulfilling whole IEEE 754 standard is required this option should be disabled.
+    Parameters
+    ----------
+    value : bool
+        State of flush-to-zero and denormals-are-zero in MXCSR register
```

Comment:
```suggestion
def set_flush_denorms(value):
    """Change floating-point calculations on CPU when dealing with denormalized values.
       This is only applicable to architectures which supports flush-to-zero.
       Denormalized values are positive and negative values that are very close to 0
       (exponent is the smallest possible value).
       Flushing denormalized values to 0 can speedup calculations if such values occurs,
       but if fulfilling whole IEEE 754 standard is required this option should be disabled.

    Parameters
    ----------
    value : bool
        State of flush-to-zero and denormals-are-zero in MXCSR register
```

Reply:
@ptrendx is there an equivalent setting for CUDA? What behavior should we expect for denormal values in GPU calculation?

Reply:
There is a `--ftz=true` compiler flag to `nvcc`, and otherwise the rounding mode is controlled per instruction, see: https://docs.nvidia.com/cuda/floating-point/index.html#rounding-modes

Reply:
So in order to implement this as runtime behavior, mxnet needs to call different functions.